client/app/components/queries/ScheduleDialog.jsx

```diff
@@ -192,7 +190,13 @@ class ScheduleDialog extends React.Component {
             <div data-testid="time">
               <TimePicker
                 allowEmpty={false}
-                defaultValue={moment().hour(hour).minute(minute)}
+                defaultValue={
+                  hour
+                    ? moment()
+                      .hour(hour)
+                      .minute(minute)
+                    : null
+                }
```

Comment:
@ranbena,

This is the actual fix. Before it was:

```javascript
defaultValue = {  moment().hour(hour).minute(minute) }
```

And now it's:

```javascript
defaultValue = { hour ? moment().hour(hour).minute(minute) : null }
```

(`moment().hour(null)` returns current hour as integer)